fuse_examples/tests/test_segmentation_picai.py

```diff
@@ -0,0 +1,75 @@
+"""
+(C) Copyright 2021 IBM Corp.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Created on June 30, 2021
+
+"""
+# FIXME: data_package
+import multiprocessing
+from fuse.utils import NDict
+import unittest
+import shutil
+import tempfile
+import os
+from fuse.utils.gpu import choose_and_enable_multiple_gpus
+from fuse.utils.multiprocessing.run_multiprocessed import run_in_subprocess
+
+import hydra
+from hydra import compose, initialize
+
+
+# os env variable PICAI_DATA_PATH is a path to the stored dataset location
+# dataset should be download from https://zenodo.org/record/6517398#.ZGEHVXZBxD8
+# labels should be downloaded from https://github.com/DIAGNijmegen/picai_labels
+# folder named PICAI which is the downloaded data folder with partition for images and labels
+if "PICAI_DATA_PATH" in os.environ:
+    from fuse_examples.imaging.segmentation.picai.runner import run_train, run_infer, run_eval
+
+
+def run_picai(root: str) -> None:
+    initialize(config_path="../imaging/segmentation/picai/conf", job_name="test_app")
+    cfg = compose(config_name="config_template")
+    print(str(cfg))
+
+    cfg = NDict(hydra.utils.instantiate(cfg))
+    cfg["paths.working_dir"] = root
```

Comment:
I think that you change your `paths.working_dir` after you instantiate the config. Thus the cache dir doesn't have the expected value: `${paths.working_dir}/cache/PICAI`.

Please double check me on that :)

Reply:
Thanks, fixed the overide to make it work properly